app/handlers/testConfig.py

```diff
@@ -47,28 +47,37 @@ def post(self):
 
     @staticmethod
     def handler(identifier, frame_start, num_frames, test_flag):
-        StatusHelper.set_status(identifier, "configuration_test", 1)
+        if test_flag == "feature":
+            status_type = Status.Type.FEATURE_TEST
+        elif test_flag == "object":
+            status_type = Status.Type.OBJECT_TEST
 
         project_path = get_project_path(identifier)
         if not os.path.exists(project_path):
-            StatusHelper.set_status(identifier, "configuration_test", -1)
+            StatusHelper.set_status(identifier, status_type, Status.Flag.FAILURE)
             return (400, 'Project directory does not exist. Check your identifier?')
 
         if test_flag == "feature":
             print "running feature"
+            if StatusHelper.get_status(identifier)[Status.Type.UPLOAD_HOMOGRAPHY] == Status.Flag.COMPLETE:
+                StatusHelper.set_status(identifier, status_type, Status.Flag.IN_PROGRESS)
+            else:
+                return (400, "Uploading homography did not complete successfully.")
```

Comment:
Maybe be a bit more verbose to as what they should do to resolve this? ie. "Uploading Homography was not completed succesfully, rerun it before proceeding"